app/helpers/application_helper.rb

```diff
@@ -1,4 +1,19 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def resource_name
```

Comment:
ApplicationHelper seems a general class. But this is something to do with user right? I think its better to make a separate class.

Reply:
Methods added here will be available for entire view classes, so generic methods like resource_name shouldn't be overrides here